docs/querying/sql.md

```diff
@@ -956,6 +955,30 @@ trailer they all include: one blank line at the end of the result set. If you de
 through a JSON parsing error or through a missing trailing newline, you should assume the response was not fully
 delivered due to an error.
 
+### HTTP DELETE
+You can use the HTTP `DELETE` method to cancel a SQL query. When you cancel a query canceled, Druid marks its `SqlLifecycle` canceled immediately. After that, any operations in `SqlLifecycle` become non-operational to avoid unnecessary execution of expensive operations.
+
+HTTP DELETE method uses the following syntax:
+```
+DELETE https://BROKER:8082/druid/v2/sql/{sqlQueryId}
```

Comment:
I like that this is clearly laid out in a code block, but it's a different format than is used to describe `HTTP POST`, which says: "You can make Druid SQL queries using HTTP via POST to the endpoint `/druid/v2/sql/`." 

Perhaps have a DELETE endpoint description similar to HTTP POST, or edit HTTP POST to have a code block like done here?